src/main/drivers/sonar_hcsr04.c

```diff
@@ -134,30 +138,33 @@ void hcsr04_init(const sonarHardware_t *initialSonarHardware, sonarRange_t *sona
     NVIC_InitStructure.NVIC_IRQChannelCmd = ENABLE;
     NVIC_Init(&NVIC_InitStructure);
 
-    lastMeasurementAt = millis() - 60; // force 1st measurement in hcsr04_get_distance()
-#else
-    lastMeasurementAt = 0; // to avoid "unused" compiler warning
 #endif
 }
 
-// measurement reading is done asynchronously, using interrupt
+#ifdef UNIT_TEST
+void digitalHi(void *p, uint16_t i) {UNUSED(p);UNUSED(i);}
```

Comment:
hcsr04_start_reading is not compiled by `#if !defined(UNIT_TEST)`, these stubs are unused